server/src-lib/Hasura/Server/Init.hs

```diff
@@ -180,13 +180,18 @@ mkServeOptions rso = do
                                 then WS.PermessageDeflateCompression WS.defaultPermessageDeflate
                                 else WS.NoCompression
                           }
+  webSocketKeepAlive <-
+    maybe
+      (Just . KeepAliveDelay $ Seconds (secondsToDiffTime 5)) -- default keep-alive interval is 5 seconds
+      (Just . KeepAliveDelay . Seconds . secondsToDiffTime)
+      <$> (withEnv (rsoWebSocketKeepAlive rso) (fst webSocketKeepAliveEnv))
```

Comment:
(I may have made a mistake here; please check my work.)
```suggestion
  webSocketKeepAlive <- Just . KeepAliveDelay . Seconds . secondsToDiffTime . fromMaybe 5
      <$> (withEnv (rsoWebSocketKeepAlive rso) (fst webSocketKeepAliveEnv))
```

Reply:
Looks fine, you just had extra `Just` and now we are modelling this thing as non-maybe value.